st2actions/st2actions/config.py

```diff
@@ -41,11 +42,19 @@ def _register_common_opts():
 
 
 def _register_action_runner_opts():
+    default_python_bin_path = sys.executable
+    base_dir = os.path.dirname(os.path.realpath(default_python_bin_path))
+    default_virtualenv_bin_path = os.path.join(base_dir, 'virtualenv')
     logging_opts = [
         cfg.StrOpt('logging', default='conf/logging.conf',
                    help='location of the logging.conf file'),
-        cfg.StrOpt('python_binary', default=sys.executable,
-                   help='Python binary which will be used by Python actions.')
+        cfg.StrOpt('python_binary', default=default_python_bin_path,
+                   help='Python binary which will be used by Python actions.'),
+        cfg.StrOpt('virtualenv_binary', default=default_virtualenv_bin_path,
+                   help='Virtualenv binary which should be used to create pack virtualenvs.'),
+        cfg.ListOpt('virtualenv_opts', default=['--always-copy', '--system-site-packages'],
```

Comment:
I also googled around a bit and it looks like `--always-copy` didn't always work correctly in some older versions of virtualenv.

Because of that we need to make sure the latest confirmed working version of virtualenv and installed and used.